CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 Changelog is rather internal in nature. See release notes for the public overview and guidelines. Releases are recorded as git tags in the [Github releases](https://github.com/learningequality/kolibri-design-system/releases) page.
 
+## Upcoming version 5.x.x (`develop` branch)
```

Comment:
Thank you, @GarvitSinghal47! I will leave the review to @nucleogenesis, just chiming in to ask if you could remove this change (probably caused by rebasing branches?)